roles/ceph-common/tasks/installs/install_on_debian.yml

```diff
@@ -0,0 +1,74 @@
+---
+- name: install dependencies
+  apt: >
+    pkg={{ item }}
+    state=present
+    update_cache=yes
+    cache_valid_time=3600
+  with_items:
+    - python-pycurl
+    - ntp
+    - hdparm
+
+- name: install the ceph repository stable key
+  apt_key: >
```

Comment:
There are definitely some style concerns. This is pulling in code from a community ceph role. The upside is a lot of this code already exists and is maintained, the downside is that it doesn't mesh with our preferred style (and there is a lot of code for Red Hatisms that we just won't hit).

I think if we mark the files we're wholly copying from upstream as coming from upstream (important for licensing issues too) we can overlook the code formatting.

On the whole, pulling in existing code kinda sucks. Yet installing the role into a central path means we're using code we don't have in our tree, which makes finding / fixing issues more difficult. A balance has to be struck.